docs/i18n.md

```diff
@@ -0,0 +1,91 @@
+# i18n
+
+## Context
+
+Internationalization on Force was introduced as a hackathon project ([PR](https://github.com/artsy/force/pull/9797)).
+
+We are using the [react-i18next](https://react.i18next.com/) internationalization framework for this.
+
+The goal of these change is separating the text content from the codebase, so copy updates can handled in a smoother process.
+
+See [this
+presentation](https://docs.google.com/presentation/d/1VT44uoGAaHX0EcDaYwGHiY8VU-fAkOXK6rSHto/edit#slide=id.gd1b1ff661f_0_12) about the benefits we expect to achieve.
+
+## How to
+
+Locales are defined on [src/System/locales](https://github.com/artsy/force/tree/main/src/System/locales). Currently, the only language supported is en-US.
+
+### Importing react-i18next
+
+1. Import the library:
+
+```jsx
+import { useTranslation } from "react-i18next”
+```
+
+2. Use the `useTranslation` hook on functional components:
+
+```jsx
+const { t } = useTranslation()
+```
+
+### Adding a static string
+
+1. Add the string to [translation.json](https://github.com/artsy/force/blob/main/src/System/locales/en-US/translation.json) following the Naming conventions.
+2. Use it in any component by calling `{t`<path-for-the-key>`}` (example: `{t`navbar.shows`}`)
+
+### Adding a dynamic string
+
+You can use [interpolation](https://www.i18next.com/translation-function/interpolation) in case you need to pass dynamic values
+
+1. Add a string to translation.json defining the keys (example: `"greetings": "hello {{name}}"`)
+2. When calling it in a component, pass the dynamic values as keys (example: `{t('hello', { name: 'John')}`)
+
+### Working with class components
+
+In some scenarios, we'll need to pass the `t` function as an argument to another function.
+We can do this by using the [ClassI18n](https://github.com/artsy/force/blob/main/src/System/i18n/ClassI18n.tsx) class.
+See an example of usage on the [SearchBar](https://github.com/artsy/force/blob/main/src/Components/Search/SearchBar.tsx). As the
+`Form` component is wrapped inside `ClassI18n` , it is capable of passing the `t` function as an argument to the `renderAutosuggestComponent` function.
+
+### Naming conventions for translations keys
+
+We always use:
+
+- The name of the component followed by the key definition in dot notation;
+- lowerCamelCase;
```

Comment:
kinda off topic, but i remember reading about that stuff, and apparently its `camelCase` (lower), and `PascalCase` (upper) 🤷

Reply:
Thanks, I pushed this change on [14ed375](https://github.com/artsy/force/pull/10588/commits/14ed3756ce79270dd517756d4ac96102e58a8add)